modules/govuk_elasticsearch/templates/es-backup-s3.erb

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -e
+
+# Redirect stdout and stderr to syslog
+exec 1> >(/usr/bin/logger -s -t $(basename $0)) 2>&1
+
+# The default Icinga passive alert assumes that the script failed
+NAGIOS_CODE=2
+NAGIOS_MESSAGE="CRITICAL: Mongodb backup push to S3 failed"
```

Comment:
There are 2 references to MongoDB in this file which should probably be Elasticsearch.